blessed/keyboard.py

```diff
@@ -168,7 +194,8 @@ def _get_modified_keycode_name(self) -> Optional[str]:
         """
         Get name for modern/legacy CSI sequence with modifiers.
 
-        Returns name like 'KEY_CTRL_ALT_F1' or 'KEY_SHIFT_UP'.
+        Returns name like 'KEY_CTRL_ALT_F1' or 'KEY_SHIFT_UP_RELEASED'. Also handles release/repeat
```

Comment:
I'm getting some inconsistent behavior with compound releases.
KEY_SHIFT_UP_RELEASED get's named as expected, but I get None for released when combined with regular characters instead of something like KEY_CTRL_S_RELEASED

Reply:
looking good,
```
'j'    '\x1b[106;5u'        'KEY_CTRL_J'              pressed CTRL
'j'    '\x1b[106;5:2u'      'KEY_CTRL_J_REPEATED'     repeated CTRL
'j'    '\x1b[106;5:2u'      'KEY_CTRL_J_REPEATED'     repeated CTRL
'j'    '\x1b[106;5:2u'      'KEY_CTRL_J_REPEATED'     repeated CTRL
'j'    '\x1b[106;5:2u'      'KEY_CTRL_J_REPEATED'     repeated CTRL
'j'    '\x1b[106;5:2u'      'KEY_CTRL_J_REPEATED'     repeated CTRL
''     '\x1b[106;5:3u'      'KEY_CTRL_J_RELEASED'     released CTRL
```